docs/kubernetes/nginx/nginx-ingress.yaml

```diff
@@ -87,9 +87,7 @@ spec:
           image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.16.2
           args:
           - /nginx-ingress-controller
-          - --configmap=$(POD_NAMESPACE)/tectonic-custom-error
```

Comment:
Should this be set explicitly to something else perhaps? Like:
```
        - --configmap=$(POD_NAMESPACE)/nginx-configuration
```

**About the command line argument:**
https://github.com/kubernetes/ingress-nginx/blob/c3ce6b892e5f1cc1066f81c19482dded2901ad45/docs/user-guide/cli-arguments.md#command-line-arguments

**Examples of this being done:**
https://github.com/kubernetes/ingress-nginx/search?utf8=%E2%9C%93&q=--configmap&type=

Reply:
That's true, but in fact there's no need to overwrite the configuration in a standard HTTP/HTTPS  + Mailu setup
So I removed it to make everything transparent. Advanced users will figure out themselves how to add extra configuration options